# reasonable parameters to find planes of @Work workspaces
voxel_leaf_size: 0.01
voxel_filter_field_name: z
voxel_filter_limit_min: -0.15
voxel_filter_limit_max: 0.3
enable_passthrough_filter: False
passthrough_filter_field_name: x
passthrough_filter_limit_min: 0.45 # all the min distance starts from the baselink 
passthrough_filter_limit_max: 0.7
enable_cropbox_filter: True
cropbox_filter_min_x: 0.40
cropbox_filter_max_x: 0.60
cropbox_filter_min_y: -0.4
cropbox_filter_max_y: 0.4
cropbox_filter_min_z: -0.1
cropbox_filter_max_z: 0.6
normal_radius_search: 0.03
use_omp: true
num_cores: 8
sac_max_iterations: 1000
sac_distance_threshold: 0.01
sac_optimize_coefficients: true
sac_x_axis: 0.0
sac_y_axis: 0.0
sac_z_axis: 1.0
sac_eps_angle: 0.09
sac_normal_distance_weight: 0.05
prism_min_height: 0.01
prism_max_height: 0.10
outlier_radius_search: 0.03
outlier_min_neighbors: 20
cluster_tolerance: 0.02
cluster_min_size: 25
cluster_max_size: 20000
cluster_min_height: 0.011
cluster_max_height: 0.09
cluster_max_length: 0.25
cluster_min_distance_to_polygon: 0.04
octree_resolution: 0.01
object_height_above_workspace: 0.0 # setting it to 0 because we reduced the gripper height
# parameters for finding empty spaces on a plane
empty_space_point_count_percentage_threshold: 0.8
empty_space_radius: 0.05
num_of_empty_spaces_required: 4
trial_duration: 3.0
enable_debug_pc: false
